docs/language/src/reference/properties.md

```diff
@@ -84,6 +84,8 @@ are marked as dirty.
 
 Callbacks in native code by default don't depend on any properties unless they query a property in the native code.
 
+The value of a property that doesn't have any bindings assigned to it is the default value of the type
```

Comment:
```suggestion
The value of a property that doesn't have any bindings assigned to it is the default value of the type.
```

That said, I think that this is the wrong location. In the beginning of the file we have this:

```markdown
You can assign values or entire
[expressions](expressions.md) to them:
```

followed by an example.

After this is example is where I would place:

```
The default value of a property is the default value of the type. For example a boolean property defaults to `false`, an `int` property to zero, etc.
```